src/registryctl/api/registry/gc/gc.go

```diff
@@ -0,0 +1,62 @@
+package gc
+
+import (
+	"bytes"
+	"github.com/goharbor/harbor/src/lib/log"
+	"github.com/goharbor/harbor/src/registryctl/api"
+	"net/http"
+	"os/exec"
+	"time"
+)
+
+// NewHandler returns the handler to handler blob request
+func NewHandler(registryConf string) http.Handler {
+	return &handler{
+		registryConf: registryConf,
+	}
+}
+
+type handler struct {
+	registryConf string
+}
+
+// ServeHTTP ...
+func (h *handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	switch req.Method {
+	case http.MethodPost:
+		h.start(w, req)
+	default:
+		api.HandleForbidden(w)
+	}
+}
+
+// Result ...
+type Result struct {
+	Status    bool      `json:"status"`
+	Msg       string    `json:"msg"`
+	StartTime time.Time `json:"starttime"`
+	EndTime   time.Time `json:"endtime"`
+}
+
+// start ...
+func (h *handler) start(w http.ResponseWriter, r *http.Request) {
+	cmd := exec.Command("/bin/bash", "-c", "registry_DO_NOT_USE_GC garbage-collect --delete-untagged=false "+h.registryConf)
```

Comment:
`registry_DO_NOT...` should be a const? and would it be better if it's absolute path?  And are you sure we need `bash` here as the file is a binary??

Reply:
But this is less important as we plan to remove it, if everything goes well, right...

Reply:
nothing changed on the command in this PR but refactor, it works fine and will be deprecated soon. so just keep it as it is?

Reply:
ok